packages/context-editor/package.json

```diff
@@ -105,8 +105,9 @@
 		"prosemirror-schema-list": "^1.5.1",
 		"prosemirror-state": "^1.4.3",
 		"prosemirror-suggest": "^3.0.0",
+		"prosemirror-tables": "^1.7.1",
 		"prosemirror-transform": "^1.10.0",
-		"prosemirror-view": "1.37.1",
+		"prosemirror-view": "1.39.2",
```

Comment:
I think we actually need this at `1.37.1` per https://github.com/handlewithcarecollective/react-prosemirror?tab=readme-ov-file#installation 

> Note: React ProseMirror releases are coupled to specific prosemirror-view releases, and are not guaranteed to work with other versions of prosemirror-view. Ensure that your version of prosemirror-view matches the version in React ProseMirror's peer dependencies!

Reply:
Hmmm okay. Everything seems to work with the bump, but I'll revert this change and silence the type errors I was getting.